File: 2018/pset6/sentimental/mario/less/mario.py
# Imported modules and functions
from cs50 import get_int

while True:  # Get 0-23 height from user
    height = get_int("Height: ")
    if height in range(24):
        break

# Initialize variable row
row = 0

for i in range(height):
    print(" " * (height - row - 1), end="")  # Print spaces
    print("#" * (row + 2), end="")  # Print bricks
    print()  # Print out i many rows
    row += 1  # Increment row

File: 2018/pset1/mario/less/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    //Initialize variables h for height and r for rows

    int h;
    int r;

    // Prompt user for height between 0 and 23

    do
    {
        h = get_int("Height: ");
    }
    while (h < 0 || h > 23);


    // Print number of rows r to match height

    for (r = 0; r < h; r++)
    {
        // Print number of spaces in each row
        for (int s = (h - r - 1); s > 0; s--)
        {
            printf(" ");
        }

        // Print number of dashes per row
        for (int d = 1; d <= (r + 2); d++)
        {
            printf("#");
        }

        printf("\n");
    }
}